src/node/rpc/common_endpoint_registry.h

```diff
@@ -7,31 +7,174 @@
 #include "http/ws_consts.h"
 #include "json_handler.h"
 #include "node/code_id.h"
+#include "node/quote.h"
+#include "node/rpc/node_interface.h"
 
 namespace ccf
 {
+  struct Quote
+  {
```

Comment:
Let's wrap this in a top-level + a version field, set to "oe_sgx_v1" or something along those lines, for future proofing.